testing/async/exclusive_scan/large_types.cu

```diff
@@ -0,0 +1,63 @@
+#include <thrust/detail/config.h>
+
+#if THRUST_CPP_DIALECT >= 2014
+
+#include <async/test_policy_overloads.h>
+
+#include <async/exclusive_scan/mixin.h>
+
+#include <unittest/special_types.h>
+
+// This test is an adaptation of TestScanWithLargeTypes from scan.cu.
+
+// Need special initialization for the FixedVector type:
+template <typename value_type>
+struct device_vector_fill
+{
+  using input_type = thrust::device_vector<value_type>;
+
+  static input_type generate_input(std::size_t num_values)
+  {
+    input_type input(num_values);
+    thrust::fill(input.begin(), input.end(), value_type{2});
+    return input;
+  }
+};
+
+template <typename value_type, typename alternate_binary_op = thrust::maximum<>>
+struct invoker
+    : device_vector_fill<value_type>
+    , testing::async::mixin::output::device_vector<value_type>
+    , testing::async::exclusive_scan::mixin::postfix_args::
+        all_overloads<value_type, alternate_binary_op>
+    , testing::async::exclusive_scan::mixin::invoke_reference::host_synchronous<
+        value_type>
+    , testing::async::exclusive_scan::mixin::invoke_async::basic
+    , testing::async::mixin::compare_outputs::assert_maybe_fuzzy_equal_quiet
+{
+  static std::string description()
+  {
+    return "scan with large value types.";
+  }
+};
+
+struct TestLargeTypes
+{
+  void operator()(std::size_t num_values) const
+  {
+    using testing::async::test_policy_overloads;
+
+    test_policy_overloads<invoker<FixedVector<int, 1>>>::run(num_values);
+
+#ifndef __QNX__ // These are excluded on QNX for scan.cu's version of this test.
```

Comment:
I'm not sure why this was failing only on QNX, but I think we should remove this ifdef and see what happens!

Reply:
Done.